src/lib/net/SecureSocket.cpp

```diff
@@ -704,18 +704,30 @@ SecureSocket::verifyCertFingerprint()
         kFingerprintDirName,
         kFingerprintTrustedServersFilename);
 
+    // Provide debug hint as to what file is being used to verify fingerprint trust
+    LOG((CLOG_NOTE "trustedServersFilename: %s", trustedServersFilename.c_str() ));
+
     // check if this fingerprint exist
     String fileLine;
     std::ifstream file;
     file.open(trustedServersFilename.c_str());
 
+    if ( ! file.is_open() ) {
```

Comment:
Spacing and indentation styles should be consistent with the style used by the rest of Barrier code.

Reply:
good catch - updated PR